Changelog.md

```diff
@@ -1,1010 +1,1010 @@
+    * [Pull request 194](https://github.com/gazebosim/gz-rendering/pull/194)
 
-### Ignition Rendering 3.3.0 (2021-01-22)
+### Gazebo Rendering 3.3.0 (2021-01-22)
 
 1. Add Windows installation.
-    * [Pull request 196](https://github.com/ignitionrobotics/ign-rendering/pull/196)
+    * [Pull request 196](https://github.com/gazebosim/gz-rendering/pull/196)
 
 1. Make flaky VisualAt test more verbose.
-    * [Pull request 174](https://github.com/ignitionrobotics/ign-rendering/pull/174)
+    * [Pull request 174](https://github.com/gazebosim/gz-rendering/pull/174)
 
 1. Resolve updated codecheck issues.
-    * [Pull request 173](https://github.com/ignitionrobotics/ign-rendering/pull/173)
+    * [Pull request 173](https://github.com/gazebosim/gz-rendering/pull/173)
 
 1. Fix crash due to NaN pose values.
-    * [Pull request 169](https://github.com/ignitionrobotics/ign-rendering/pull/169)
+    * [Pull request 169](https://github.com/gazebosim/gz-rendering/pull/169)
 
 1. Improve fork experience.
-    * [Pull request 165](https://github.com/ignitionrobotics/ign-rendering/pull/165)
+    * [Pull request 165](https://github.com/gazebosim/gz-rendering/pull/165)
 
-### Ignition Rendering 3.2.0 (2020-10-13)
+### Gazebo Rendering 3.2.0 (2020-10-13)
 
 1. Add Custom Render Engine support
-    * [Pull request 161](https://github.com/ignitionrobotics/ign-rendering/pull/161)
-    * [Pull request 154](https://github.com/ignitionrobotics/ign-rendering/pull/154)
-    * [Pull request 142](https://github.com/ignitionrobotics/ign-rendering/pull/142)
-    * [Pull request 141](https://github.com/ignitionrobotics/ign-rendering/pull/141)
+    * [Pull request 161](https://github.com/gazebosim/gz-rendering/pull/161)
+    * [Pull request 154](https://github.com/gazebosim/gz-rendering/pull/154)
+    * [Pull request 142](https://github.com/gazebosim/gz-rendering/pull/142)
+    * [Pull request 141](https://github.com/gazebosim/gz-rendering/pull/141)
 
 1. Add getter for Ogre2Camera
-    * [Pull request 97](https://github.com/ignitionrobotics/ign-rendering/pull/97)
+    * [Pull request 97](https://github.com/gazebosim/gz-rendering/pull/97)
 
 1. Update test configuration to use ogre2
-    * [Pull request 83](https://github.com/ignitionrobotics/ign-rendering/pull/83)
+    * [Pull request 83](https://github.com/gazebosim/gz-rendering/pull/83)
 
 1. Ogre2 GPU Ray: Cleanup all resources on destruction
-    * [BitBucket pull request 258](https://osrf-migration.github.io/ignition-gh-pages/#!/ignitionrobotics/ign-rendering/pull-requests/258)
+    * [BitBucket pull request 258](https://osrf-migration.github.io/ignition-gh-pages/#!/gazebosim/gz-rendering/pull-requests/258)
 
 1. Ogre2: Add Grid
-    * [BitBucket pull request 246](https://osrf-migration.github.io/ignition-gh-pages/#!/ignitionrobotics/ign-rendering/pull-requests/246)
+    * [BitBucket pull request 246](https://osrf-migration.github.io/ignition-gh-pages/#!/gazebosim/gz-rendering/pull-requests/246)
```

Comment:
Need to revert the `osrf-migration` changes

Reply:
https://github.com/gazebosim/gz-rendering/pull/629/commits/232678d5ac0decbeb41f6acd8a9d35e3a79a78e9